src/test/java/com/snack/news/service/AdminServiceTest.java

```diff
@@ -111,4 +117,29 @@ void deleteNewsTestWhenNotExistNewsId() {
 		doThrow(new IllegalArgumentException()).when(newsRepository).deleteById(anyLong());
 		assertThrows(NewsNotFoundException.class, () -> adminService.deleteNews(anyLong()));
 	}
+
+
+	@ParameterizedTest
+	@DisplayName("Admin에서 pick 리스트를 페이지별로 조회할 수 있다")
+	@ValueSource(ints = {1, Integer.MAX_VALUE})
+	void getPickListTestOrderByDate(final int page) {
+		adminService.getPickPage(page);
+		verify(picksRepository).findAll(any(Pageable.class));
+	}
+
+	@ParameterizedTest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)